src/tests/ftest/util/soak_utils.py

```diff
@@ -429,19 +430,26 @@ def launch_vmd_identify_check(self, name, results, args):
     else:
         uuids = uuid_list
     self.log.info("VMD device UUIDs: %s", uuids)
+    host_uuids = get_storage_query_device_uuids(self.dmg_command)
+    for host, uuid_dict in host_uuids.items():
+        uuid_list = sorted(uuid_dict.keys())
+        self.log.info("Devices on hosts %s: %s", host, uuid_list)
```

Comment:
host or hosts?

Reply:
It's just host... Need to fix the self.log.info message.